src/gbastd/types/time.h

```diff
@@ -0,0 +1,36 @@
+#ifndef TIME_H
+#define TIME_H
+
+#include "u.h"
+#include "../../defines.h"
+
+typedef struct {
+    u8 seconds;
+    u8 minutes;
+    u8 hours;
+
+    u64 tick;
+} time;
+
+void addTick(time* time) {
+    if (time->tick + 1 < U64_LIMIT)
+        time->tick++;
```

Comment:
So I guess, timer will just stop running, after hitting limit. This is not exactly what we want. And thy addTick accepts *time* and not a Timer struct ?